repository/PatientRepository.js

```diff
@@ -22,6 +33,15 @@ class PatientRepository {
   fetchPatientStatusForms(userId) {
     return this.mongo.db('test').collection('patientForms').find({patientUid: userId}).toArray();
   }
+
+  raiseFlag(userId, flagType, flagValue) {
+    return this.mongo.db('test').collection('user').updateOne({uid: userId}, {$set: {flagType: flagValue}});
+  }
+
+  fetchPatientsCovidInfo() {
+    return this.mongo.db('test').collection('user').find({userType: 'patient', userStatus: 'APPROVED'}, {_id: 0, uid: 1, covidStatus: 1, name: 1}).toArray();
```

Comment:
Add covidStatus to the **_data_** inside route /addnewUser and I think we're good